bin/alluxio

```diff
@@ -81,7 +81,7 @@ EOF
 function killAll {
   if [[ $# -ne 1 ]]; then
     echo "Usage: alluxio killAll <WORD>"
-    exit
+    exit 1
```

Comment:
https://github.com/Alluxio/alluxio/blob/837a41c68b956ded56ae0fd41f0bf26b52ffbf49/bin/alluxio#L112
but
https://github.com/Alluxio/alluxio/blob/837a41c68b956ded56ae0fd41f0bf26b52ffbf49/bin/alluxio#L227
and in file `integration/emr/alluxio-emr.sh` all exit 2.
Which one is better?@LuQQiu

Reply:
perhaps exit 2 is better here
according to https://tldp.org/LDP/abs/html/exitcodes.html